### PR TITLE
style: format code

### PR DESCRIPTION
**Pull request by yrong**

### CODE REVIEW

File: core/packages/contracts/.prettierrc [hunk 1]
```diff
@@ -7,7 +7,8 @@
           "tabWidth": 4,
           "singleQuote": false,
           "bracketSpacing": true,
-          "explicitTypes": "always"
+          "explicitTypes": "always",
+          "plugins": ["prettier-plugin-solidity"]
```
Comment (vgeddes) [inline core/packages/contracts/.prettierrc:11]: nice!

**Review by vgeddes — COMMENTED** [review]
Thanks! Though you need to refresh the PR since https://github.com/Snowfork/snowbridge/pull/735 introduced some conflicts.

File: core/packages/contracts/.prettierrc (hunk 1 above)
Comment (yrong) [reply core/packages/contracts/.prettierrc:11]: @vgeddes  conflict resolved, no logic but only format change in this pr and also add a git hook for it when commit

File: .husky/pre-commit [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+cd core && pnpm format
```
Comment (vgeddes) [inline .husky/pre-commit:4]: Nice! Can we also make the hook fail if `pnpm lint` fails?

**Review by vgeddes — APPROVED** [review]
🚀

**Review by vgeddes — APPROVED** [review]
+1

**Review by alistair-singh — APPROVED** [review]

**Pull request merged**